src/actions/breakpoints.js

```diff
@@ -36,17 +44,55 @@ async function _getGeneratedLocation(source, sourceMaps, location) {
   return await sourceMaps.getGeneratedLocation(location, source.toJS());
 }
 
+async function syncClientBreakpoint(
```

Comment:
minor style nit, but i move actiony things to options i.e.

`syncClientBreakpoint(source, pendingBreakpoint, location, {client, sourceMaps})`